frappe/templates/includes/login/login.js

```diff
@@ -7,6 +7,13 @@ window.login = {};
 
 window.verify = {};
 
+
+(function foo(){
```

Comment:
why foo? what does this pattern do? Please also add some comments!

Reply:
this I forgot to remove. If the flow is correct then will update it and send a PR